pkgs/by-name/ve/vep/package.nix

```diff
@@ -0,0 +1,132 @@
+# Instructions:http://www.ensembl.org/info/docs/api/api_installation.html
+# Do not use https://github.com/Ensembl/ensembl-vep/archive/release/${version}.zip
+# We cannot use INSTALL.pl but it’s not that bad:install the dependencies and copies the .pm files should be ok
+{ config
+, curl
+, lib
+, perlPackages
+, pkgs
+, stdenv
+, fetchFromGitHub
+, git
+, perl
+, unzip
+, makeWrapper
+, which
+, zlib
+}:
+
+with pkgs;
+
+let
+  customInstallPhase = ''
+    mkdir -p $out/${perl.libPrefix}/${perl.version}/
+    tests=$(find modules/ -mindepth 1 -maxdepth 1 -type d | grep -v t)
+    cp -r $tests $out/${perl.libPrefix}/${perl.version}/
+  '';
+
+  ensemblGit = n: r: s:
+    # Copy modules directly
+    stdenv.mkDerivation rec {
+      name = n;
+      version = "110";
```

Comment:
If using an unreleased commit, the version should be `0-unstable-YYYY-MM-DD` (see https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-naming).